Fill `--repository-path` when `solution-path` is used

BenvdStouwe opened the pull request:
So it doesn't throw an exception when `solution-path` is not in the git root directory.
Fixes #237

leonardochaia commented:
Hi @BenvdStouwe , thanks for the PR.

Would you kindly take a look at the build error?

Thank you!!
Leo

BenvdStouwe commented:
Hi @leonardochaia, I think I fixed the problem but the Action seems to do something different in my account than it does in yours. Can you run the Action again here to see if I fixed it?

BenvdStouwe commented:
Hi @leonardochaia, could you please run the actions again?
